engineapi/system_backup.go

```diff
@@ -0,0 +1,116 @@
+package engineapi
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+
+	systembackupstore "github.com/longhorn/backupstore/systembackup"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/types"
+
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+type BackupTargetClientSystemBackupInterface interface {
+	SystemBackupConfigGet(name, version string) (*systembackupstore.Config, error)
+	SystemBackupDelete(systemBackup *longhorn.SystemBackup) (string, error)
+	SystemBackupDownload(name, version, downloadPath string) error
+	SystemBackupList() (systembackupstore.SystemBackups, error)
+	SystemBackupUpload(name, localFile, longhornVersion, longhornGitCommit, managerImage, engineImage string) (string, error)
```

Comment:
Please make the method starting with Verb. 

`GetSystemBackupConfig`, `DeleteSystemBackup`, etc.

Also, suggest renaming `BackupTargetClientSystemBackupInterface` to `SystemBackupOperationInterface`.

Reply:
Updated.